### DISCUSSION TITLE
Shapefile FIELD value surviving import? Geometry Nodes

### DISCUSSION THREAD

**Discussion by lvpvsinfabvla** [domlysz/BlenderGIS#756] [Q&A]
Hi. Are FIELD value of a shapefile exported from Qgis (eg. osm_type, natural, name, ele for some peaks OSM data) preserved and accessible after BlenderGIS import? I'd like to use these values in the Geometry Nodes to quickly generate individual geometry displaying name and elevation for each OSM peak point for a certain area over a 3D map. It could be possible?

**Comment by domlysz** [comment]
yep as object custom properties
https://github.com/domlysz/BlenderGIS/wiki/Shapefile-import
